### PR TITLE
Add support for UUID fields.

### PR DESCRIPTION
**Pull request by anivanovic**
Initial try at adding support for generating UUID fields. 
cc @a8m

### CODE REVIEW

File: schemast/printer_test.go [hunk 1]
```diff
@@ -88,6 +89,22 @@ func TestPrintHeaderComment(t *testing.T) {
...
+	uuidImportRegex := regexp.MustCompile("github.com/google/uuid")
+
+	require.NoError(t, tt.ctx.AppendField("Message", field.UUID("uuid", uuid.UUID{}).Descriptor()))
+	require.NoError(t, tt.ctx.AppendField("Message", field.UUID("hash", uuid.UUID{}).Descriptor()))
+	require.NoError(t, tt.print())
+
+	contents, err := ioutil.ReadFile(filepath.Join(tt.schemaDir(), "message.go"))
+	require.NoError(t, err)
+	matches := uuidImportRegex.FindAllString(string(contents), -1)
+	require.Len(t, matches, 1)
```
Comment (a8m) [inline schemast/printer_test.go:105]: `string.Count` instead?

https://pkg.go.dev/strings#Count

**Review by a8m — APPROVED** [review]
Looks great to me, but it's handling UUID fields and not JSON fields.

@rotemtam, any thoughts?

**Comment by a8m** [comment]
Thanks for the contribution @anivanovic, and welcome to the ent community ❤️

**Review by rotemtam — COMMENTED** [review]
Well done! Do you mind applying @a8m 's suggestion? Otherwise we're good to go.

**Comment by anivanovic** [comment]
It was a long day and my brain replaced the two. I will update PR.

**Comment by anivanovic** [comment]
Renamed branch and abandoned this PR. New one is [here](https://github.com/ent/contrib/pull/225)

**Pull request closed without merging**